dask_jobqueue/core.py

```diff
@@ -206,6 +207,8 @@ def __init__(self,
             env_extra = dask.config.get('jobqueue.%s.env-extra' % self.scheduler_name)
         if log_directory is None:
             log_directory = dask.config.get('jobqueue.%s.log-directory' % self.scheduler_name)
+        if shebang is None:
```

Comment:
I'd reconsider adding a kwarg for this.  In my understanding, kwargs are for things you change frequently.  This looks like an issue that you have to sort out once for your deployment and then never touch it again.  Just adding it to the config should cover all cases.

Reply:
I'm not sure, and I have no strong feeling on this. Currently it looks like we have kind of a match between kwargs and config file. I'm not sure which way we should go.

Reply:
I did it this way only because that's how everything else in the yaml's are set.  I don't really mind either way.  maybe let me know @guillaumeeb ?

Reply:
Yeah, that is what I thought. If so, leave it as is, as the everything else is done. Maybe we'll need to check whether all the kwargs are justified or not in the future.